.zenodo.json

```diff
@@ -51,6 +41,32 @@
             "orcid": "0000-0002-0922-5770"
         }
     ],
+    "contributors": [
+        {
+            "affiliation": "Scientific Software Engineering Center at JHU",
+            "name": "Edward Hunter",
+            "orcid": "0000-0002-6876-001X",
+            "type": "ProjectManager"
+        },
+        {
+            "affiliation": "Scientific Software Engineering Center at JHU",
+            "name": "Gerard Lemson",
+            "orcid": "0000-0001-5041-2458",
+            "type": "ProjectManager"
+        },
+        {
+            "affiliation": "Scientific Software Engineering Center at JHU",
+            "name": "Galen Mayfield",
+            "orcid": "0009-0009-6192-8623",
+            "type": "ProjectManager"
+        }
+    ],
+    "grants": [
+        {"": ""}
```

Comment:
@ehunter68 Would you be able to enquire with Schmidt about whether we can ref them as the grant source, please. It looks like zenodo accepts only OpenAIRE-supported grants for which I'm not sure if we're applicable. Here is the ``grants`` excerpt from the [zenodo docs](https://developers.zenodo.org/#representation):
>List of OpenAIRE-supported grants, which have funded the research for this deposition. Each array element is an object with the attributes:
>* id: grant ID.
>
>Example: [{'id':'283595'}] (European Commission grants only)
or funder DOI-prefixed: [{'id': '10.13039/501100000780::283595'}] (All grants, recommended)
>
>Accepted funder DOI prefixes:
Academy of Finland: 10.13039/501100002341
Agence Nationale de la Recherche: 10.13039/501100001665
Aligning Science Across Parkinson’s: 10.13039/100018231
Australian Research Council: 10.13039/501100000923
Austrian Science Fund: 10.13039/501100002428
Canadian Institutes of Health Research: 10.13039/501100000024
European Commission: 10.13039/501100000780
European Environment Agency: 10.13039/501100000806
Fundação para a Ciência e a Tecnologia: 10.13039/501100001871
Hrvatska Zaklada za Znanost: 10.13039/501100004488
Institut National Du Cancer: 10.13039/501100006364
Ministarstvo Prosvete, Nauke i Tehnološkog Razvoja: 10.13039/501100004564
Ministarstvo Znanosti, Obrazovanja i Sporta: 10.13039/501100006588
National Health and Medical Research Council: 10.13039/501100000925
National Institutes of Health: 10.13039/100000002
National Science Foundation: 10.13039/100000001
Natural Sciences and Engineering Research Council of Canada: 10.13039/501100000038
Nederlandse Organisatie voor Wetenschappelijk Onderzoek: 10.13039/501100003246
Research Councils: 10.13039/501100000690
Schweizerischer Nationalfonds zur Förderung der wissenschaftlichen Forschung: 10.13039/501100001711
Science Foundation Ireland: 10.13039/501100001602
Social Science Research Council: 10.13039/100001345
Templeton World Charity Foundation: 10.13039/501100011730
Türkiye Bilimsel ve Teknolojik Araştırma Kurumu: 10.13039/501100004410
UK Research and Innovation: 10.13039/100014013
Wellcome Trust: 10.13039/100004440